packages/core/core/src/services/event-hub.ts

```diff
@@ -92,7 +93,15 @@ export default function createEventHub(): EventHub {
     },
 
     removeAllListeners() {
-      return eventHub.destroy();
+      listeners.clear();
+      return this;
+    },
+
+    removeAllSubscribers() {
+      subscribers.length = 0;
+      // Re-add the default subscriber
+      subscribers.push(defaultSubscriber);
```

Comment:
if you have other stuff to change, you could just do a subscribers.splice(..) to keep the 1st element in one go